diff --git a/mifalim-app/tailwind.config.js b/mifalim-app/tailwind.config.js
index 1f7ff3f..f7fae1a 100644
--- a/mifalim-app/tailwind.config.js
+++ b/mifalim-app/tailwind.config.js
@@ -1,6 +1,6 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
-  content: ['./app/**/*.{js,jsx}'],
+  content: ['./app/**/*.{js,jsx}', './components/**/*.{js,jsx}', './lib/**/*.{js,jsx}'],
   theme: { extend: {} },
   plugins: [],
 };
